wire-runtime-swift/src/main/swift/Redactable.swift

```diff
@@ -44,16 +44,20 @@ extension Redactable {
             guard let label = label else {
                 return "\(value)"
             }
-            if RedactedKeys(rawValue: label) != nil {
+            var strippedLabel = label
```

Comment:
Test caught this now because we emit `@Defaulted` identity types for many properties which now as an empty string exacerbated this issue which is now fixed and the test passes.